draft-ietf-quic-recovery.md

```diff
@@ -1151,7 +1158,8 @@ OnLossDetectionTimeout():
     DetectLostPackets(pn_space)
   // Retransmit crypto data if no packets were lost
   // and there are still crypto packets in flight.
-  else if (crypto packets are in flight):
+  else if (crypto packets are in flight &&
+           crypto data to send):
```

Comment:
I still don't get this. If crypto packets are in flight, doesn't that mean there is unacked crypto data to send? Why is this second predicate useful?

Reply:
Crypto packets could be in flight because of a timer based retransmission, but the original was ACked.

Reply:
See issue #1964

Reply:
Ah, ok. How about just "unacked crypto data exists"? Is that enough to capture it?

Reply:
I meant using just this one condition instead of the two that you have.

Reply:
I agree that this wording is confusing; in particular, it's not obvious that "to send" means "not yet acknowledged," except perhaps by process of elimination.

Reply:
Sure, I took Jana's suggestion.